src/store/index.ts

```
@@ -254,6 +255,8 @@ export const useBikeTagStore = defineStore(BikeTagDefaults.store, {
       const initResults: any[] = []
       this.fetchingData = true
 
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)